jsonschema-core.md

```diff
@@ -2081,7 +2082,7 @@ consumers will need to properly interpret validation results.
 JSON Schema output is defined using the JSON Schema data instance model as
 described in section 4.2.1. Implementations MAY deviate from this as supported
 by their specific languages and platforms, however it is RECOMMENDED that the
-output be convertible to the JSON format defined herein via serialization or
+output be convertib le to the JSON format defined herein via serialization or
```

Comment:
```suggestion
output be convertible to the JSON format defined herein via serialization or
```

Reply:
Nice catch, yeah.  The fix for that came in when I rebased just now.